pkg/network/usm/monitor_test.go

```diff
@@ -583,30 +579,16 @@ func requestGenerator(t *testing.T, targetAddr string, reqBody []byte) func() *n
 	}
 }
 
-func includesRequest(t *testing.T, allStats map[http.Key]*http.RequestStats, req *nethttp.Request) {
-	expectedStatus := testutil.StatusFromPath(req.URL.Path)
-	included, err := isRequestIncludedOnce(allStats, req)
-	require.NoError(t, err)
-	if !included {
-		t.Errorf(
-			"could not find HTTP transaction matching the following criteria:\n path=%s method=%s status=%d",
-			req.URL.Path,
-			req.Method,
-			expectedStatus,
-		)
-	}
-}
-
-func requestNotIncluded(t *testing.T, allStats map[http.Key]*http.RequestStats, req *nethttp.Request) {
+func checkRequestIncluded(t *testing.T, allStats map[http.Key]*http.RequestStats, req *nethttp.Request, expectedToBeIncluded bool) {
 	included, err := isRequestIncludedOnce(allStats, req)
 	require.NoError(t, err)
-	if included {
-		expectedStatus := testutil.StatusFromPath(req.URL.Path)
+	if included != expectedToBeIncluded {
 		t.Errorf(
-			"should not find HTTP transaction matching the following criteria:\n path=%s method=%s status=%d",
+			"%s not find HTTP transaction matching the following criteria:\n path=%s method=%s status=%d",
```

Comment:
the begging of the error message does not sounds right.

Reply:
talked by heart